diagnostic.md

```diff
@@ -3,36 +3,38 @@
 What are three advantages of using Ember?
 
 ```sh
-# your answer here
+Adding routes to SPAs.
+Adding views to SPAs.
+Dynamic update of page content.
 ```
 
 What is the templating system used in Ember? How it is related to
 handlebars?
 
 ```sh
-# your answer here
+The templating system in Ember is pretty much just handlebars. They were quite possibly worked on by the same person.
```

Comment:
They were in fact worked on by the same person!
But Ember handlebars is more like handlebars+. It contains added functionality including built-in helpers like `link-to`, as well as the ability to plug in other components and templates.

.5/1